tests/OpenApi.Extensions.Tests/IntegrationTests.Schema_Is_Correct_For_Sample.DotNet9_0.verified.txt

```diff
@@ -263,6 +263,56 @@
           }
         }
       }
+    },
+    /api/items/find: {
+      get: {
+        tags: [
+          TodoApp
+        ],
+        summary: Searches for Todo item by given filter.,
+        operationId: FindTodo,
+        parameters: [
+          {
+            name: Text,
+            in: query,
+            description: Gets or sets the text of filter.,
```

Comment:
Looks like this is highlighting a bug, as [this filter](https://github.com/martincostello/openapi-extensions/blob/3874a485ffe87e8bfc69966963228b06385eef14/src/OpenApi.Extensions/Transformers/DescriptionsTransformer.cs#L65-L84) should be removing the `Gets or sets` prefixes on these two properties.